301-new-vnet-peer-aadds/README.md

```diff
@@ -0,0 +1,18 @@
+# Deploy a HDInsight HBase replication across two regions
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F301-new-vnet-peer-aadds%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F301-new-vnet-peer-aadds%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template creates a new VNET with default subnet, and peers it to the existing VNET in which your Azure Active Directory Domain Services is located.
+
+## Related templates
+
+Coming soon...
+
+## Prerequisites
+
+This template assumes you have already configured Azure Active Directory Domain Services and that it is in an existing VNET.
```

Comment:
how do I create one? any reason we can't add a prereq template for it?
See: https://github.com/Azure/azure-quickstart-templates#template-pre-requisites